inc/datahandlers/user.php

```diff
@@ -868,6 +868,31 @@ function verify_language()
 	}
 
 	/**
+	 * Verifies if a style is valid for this user or not.
+	 *
+	 * @return boolean True when valid, false when invalid.
+	 */
+	function verify_style()
+	{
+		global $lang;
+
+		$user = &$this->data;
+
+		if($user['style'] && !defined('IN_ADMINCP'))
```

Comment:
If I change the style of X user from the ACP shouldn't I be allowed to do so? I feel that is how the users edit module works. Go and try it.

I may be incorrect or have the incorrect impression.